components/dashboard/src/teams/Members.tsx

```diff
@@ -188,7 +211,16 @@ export default function() {
                     </Item>)}
             </ItemsList>
         </div>
-        {genericInvite && showInviteModal && <Modal visible={true} onClose={() => setShowInviteModal(false)}>
+        {genericInvite && showInviteModal && <Modal visible={true} onEnter={() => {
```

Comment:
**question**: Since we'll track the button that triggers this modal, does it help to track if the modal opened?

Reply:
@gtsiolis Generally, I am tracking two different things: 1) What buttons were clicked and where they are positioned (e.g. `dashboard_button_clicked`) and 2) the effects on the creation / alteration / deletion on other entities the user interaction has had. In this instance, I want to track the specific `inviteUrl` as this will allow to match it up with users who follow the invitation. The URL is not a general property of a button though, so I put it in a dedicated event. As the pressing of the button will inevitably cause the URL to be displayed, I moved the call to track the request for the `inviteUrl` to the onClick event that will open the modal and removed the `dashboard_button_clicked` event from there.